script_library/run-ansible.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 
-
+# exit if vars is not set
+socok8s_absolute_dir=${socok8s_absolute_dir:?}
```

Comment:
I was kinda abusing this by sourcing that script and be able to run the plays directly, so it means I can't do it anymore?

Shouldn't that be warned with a message? It seems failing without explaining why is unfriendly...

Reply:
I guess you can as long as you set the `socok8s_absolute_dir` env manually?

Agreed on no message, let me add one to all the instances of this so its more user friendly